src/pages/VerifyEmailPage/index.tsx

```diff
@@ -11,28 +12,50 @@ import RegularText from 'src/components/shared/RegularText';
 import LabelText from 'src/components/shared/LabelText';
 import TextButton from 'src/components/shared/TextButton';
 import ButtonText from 'src/components/shared/ButtonText';
+import OTPInput from 'src/components/shared/OTPInput';
 import useVerificationTimer from './VerifyEmailPageHooks';
 
 function VerifyEmailPage() {
+  const otpLengthMin = 0;
+  const otpLengthMax = 6;
```

Comment:
move them out of the component, we don't need reinitialisation with each rerender